src/input/Search.js

```diff
@@ -18,8 +18,18 @@ class Search extends Component {
   }
 
   clearText() {
-    const ref = this.props.textInputRef;
-    this.refs[ref].clear();
+    if (this.props.onChangeText) {
+      this.props.onChangeText('');
```

Comment:
If i am thinking about this right, it's this just selling value to an empty string?